packages/builders/scripts/testing/recorded-retired-instances.js

```diff
@@ -0,0 +1,72 @@
+import { makeTracer } from '@agoric/internal';
+import { E } from '@endo/far';
+
+const trace = makeTracer('RecordedRetired', true);
+
+/**
+ * @param {BootstrapPowers &
+ *   PromiseSpaceOf<{ retiredContractInstances: MapStore<string, Instance>;
+ *   }>
+ * } powers
+ */
+export const testRecordedRetiredInstances = async ({
+  consume: {
+    contractKits,
+    // governedContractKits,
+    retiredContractInstances: retiredContractInstancesP,
+  },
+}) => {
+  trace('Start');
+  const retiredContractInstances = await retiredContractInstancesP;
+
+  const auctionIDs = Array.from(retiredContractInstances.keys()).filter(k =>
+    k.startsWith('auction'),
+  );
+  assert(auctionIDs);
+  assert(auctionIDs.length === 1);
+  const auctionInstance = retiredContractInstances.get(auctionIDs[0]);
+  // I don't know why it's neither in governedContractKits nor contractKits
+  // assert(await E(governedContractKits).get(auctionInstance));
```

Comment:
I draw your attention to this inability, but I'm going to merge it anyway. The old auction instance has been stored, but I haven't been able to look it up in in `governedContractKits` or `contractKits`. the `economicCommitttee` instance is in `contractKits`.

Reply:
kits typically make it into `governedContractKits` by way of `startGovernedUpgradable`, but `startAuctioneer` pre-dates that. It calls `E(zoe).startInstance(contractGovernorInstallation, ...)` directly.